.pylintrc_utils

```diff
@@ -0,0 +1,546 @@
+[MASTER]
```

Comment:
IMO, this shroud be split into two commits, one for pylint configuration introduction and second for addressing W1203. But that is just my opinion, I don't see it as a blocker for PR, I will keep it up to you.